oam/install_scripts/pre-uninstall

```diff
@@ -141,26 +133,28 @@ if [ $quiet != 1 ]; then
 	rm -f $installdir/etc/AlarmConfig.xml.installSave
 fi
 
-#umount file systems
-if [ $user = "root" ]; then
-	rm -f $installdir/data*/OAMdbrootCheck > /dev/null 2>&1
-else
-	$SUDO rm -f $installdir/data*/OAMdbrootCheck > /dev/null 2>&1
-fi
+#remove OAMdbrootCheck file
+rm -f $installdir/data*/OAMdbrootCheck > /dev/null 2>&1
 
 #remove library paths
 if [ $user = "root" ]; then
         rm -f /etc/ld.so.conf.d/columnstore.conf
         ldconfig
 else
-	$SUDO rm -f /etc/defaults/columnstore
-fi
+	rm -f /etc/default/columnstore
```

Comment:
This wouldn't work unless we managed to install the script there as not-root, which probably/hopefully didn't happen.